puppet/hiera/common.yaml

```diff
@@ -21,6 +21,28 @@ uber::config::event_name:               'SuperRad CoolCon'
 uber::config::organization_name:        'RadDudettes INC'
 uber::config::year:                     33
 
+uber::config::initial_attendee: 50    # badge price
+uber::config::dealer_badge_price: 30
+uber::config::max_dealers: 20
+uber::config::default_table_price: 350
+
+uber::config::table_prices:
+  1: 125
+  2: 175
+  3: 250
+  4: 350
+
+uber::config::badge_prices:
+  single_day:
+    - { 'Friday': 65 }
+    - { 'Saturday': 65 }
+    - { 'Sunday': 65 }
+
+  attendee:
+    - { '2019-09-01': 50 }
+    - { '2019-01-20': 75 }
+
+uber::config::group_prereg_takedown: '2099-08-30'
```

Comment:
I like how it's August 30 on 2099 instead of January 1 or December 31 of 2099 :P